js/controller.js

```diff
@@ -2149,14 +2149,16 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
     rewindOrRequestPreviousVideo: function() {
       var currentTimestamp = Utils.getCurrentTimestamp();
       var timeElapsed = currentTimestamp - this.state.skipControls.requestPreviousTimestamp;
-      // Button has been clicked once more in a short amount of time or playhead
-      // is below a certain threshold, request previous video.
+      // Button has been clicked once more in a short amount of time, playhead
+      // is below a certain threshold, or player is stalled after seeking, request previous video.
```

Comment:
If the user seeks waaay forward via the control bar, and then starts seeking backwards rapidly via backward button into available but unbuffered content, is it possible for the user to go back to the previous video? If so, I think this is undesirable, right?

Reply:
This applies to the "Back" button only, not the "Seek back" button. It is expected for the Back button to take you to the previous video if you click on it more than once

Reply:
How come the back button is affected by the current seeking state of the video? Just curious

Reply:
This is because the first click rewinds the video, but the playhead might take some time to update. This check ensures that you move to the previous video when clicking again, even if the playhead is not below the threshold.